src/Jebediah/Data.hs

```diff
@@ -74,6 +75,11 @@ data Query =
   | At Sequence
     deriving (Eq, Show)
 
+data StreamOutput =
+    StreamsLatest
+  | StreamsPrefix (Maybe LogStream)
```

Comment:
Does this still need the Maybe?

Reply:
For the case where someone wants the behaviour the CLI currently defaults to, list all streams by ID without filtering. I'm not sure if anyone's actually using it, it seems to hit ratelimiting more often than not.